extra/mariabackup/aria_backup_client.cc

```diff
@@ -400,16 +400,18 @@ bool Table::copy(ds_ctxt_t *ds, bool is_index, unsigned thread_num) {
 		for (ulonglong block= 0 ; ; block++) {
 			size_t length = m_cap.block_size;
 			if (is_index) {
-				if ((error= aria_read_index(
-							partition.m_index_file, &m_cap, block, copy_buffer) ==
-							HA_ERR_END_OF_FILE))
-					break;
+				error= aria_read_index(partition.m_index_file,
+						       &m_cap, block,
+						       copy_buffer);
 			} else {
-				if ((error= aria_read_data(
-							partition.m_data_file, &m_cap, block, copy_buffer, &length) ==
-							HA_ERR_END_OF_FILE))
-					break;
+				error= aria_read_data(partition.m_data_file,
+						      &m_cap, block,
+						      copy_buffer, &length);
 			}
+
+			if (error == HA_ERR_END_OF_FILE)
+			  break;
+
 			if (error) {
 				msg(thread_num, "error: aria_read %s failed:  %d",
 					is_index ? "index" : "data", error);
```

Comment:
@Thirunarayanan, can you check if fd39c63b416c4c4f8bfff6c3935fda159a25faee fixes this problem that you observed? It is not adding any special handling of `HA_ERR_END_OF_FILE` here.

Reply:
Given commit does handle HA_ERR_END_OF_FILE inside if condition. It misses some code formatting rules.